Watch Extension/HomeView.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+
+/*
+ TODO - V1 Watch
+ - Polish UI
+ - Make running feedings a live list
+ - Make sure a backgrounded/terminated main app
+   behaves well from a UI standpoint when using the watch
+ - App icon
+ - Think about subscriptions or IAP for this
+ - Better formalize communication instead of basing it off
+   of yucky strings
+ - Add unidirectional flow architecture for better state management
+ */
+
+struct HomeView: View {
+    @State private var isShowingSheet = false
+    var body: some View {
+        VStack {
+            // TODO: fill this in with active feedings
+            List((1...10), id: \.self) { num in
+                Text("00:00:00")
+            }
+            // TODO: better style this
+            // TODO: think more about the layout of these two
+            Image(systemName: "xmark.circle.fill")
+                .font(.system(size: 55))
+                .rotationEffect(.init(degrees: 45))
+                .offset(y: -10)
+                .sheet(isPresented: $isShowingSheet) {
+                    AddFeedingView(isShowingSheet: self.$isShowingSheet)
+                }
+                .gesture(TapGesture().onEnded {
+                    self.isShowingSheet.toggle()
+                })
+        }
+        .edgesIgnoringSafeArea(.bottom)
+    }
+}
+
+struct HomeView_Previews: PreviewProvider {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c49e444d61917df61aa42d9cccb11a46857e6fb9/Watch Extension/HomeView.swift#L41"></span>Type name should only contain alphanumeric characters: ‘HomeView_Previews’<br />
<code>type_name</code> <code>HomeView.swift:41</code></td>
    </tr>
  </tbody>
</table>